src/AreaView/AreaZeroData.tsx

```diff
@@ -15,8 +17,11 @@ export class AreaZeroData extends React.Component<{}, {}> {
             secondaryText={zeroDataSecondaryText}
             actionText={zeroDataActionText}
             actionType={ZeroDataActionType.ctaButton} 
-            onActionClick={(event, item) =>
-                alert("TODO - open: " + item!.actionText)
+            onActionClick={(event, item) => {
```

Comment:
i get this error
```
Uncaught Invariant Violation: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
    at invariant (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:23506:15)
    at Object.throwInvalidHookError (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:36318:3)
    at Object.useContext (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:46243:21)
    at Object../src/StateMangement/StateProvider.tsx.exports.useStateValue (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:51144:58)
    at AreaZeroData../src/AreaView/AreaZeroData.tsx.AreaZeroData.render (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:49067:48)
    at finishClassComponent (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:38192:31)
    at updateClassComponent (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:38147:24)
    at beginWork (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:39095:16)
    at performUnitOfWork (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:42763:12)
    at workLoop (https://edwon.gallerycdn.vsassets.io/extensions/edwon/okr-hub-dev/1.0.368/1564592396877/dist/OKRHub.js:42803:24)```

Reply:
Yes this is an easy fix! This function needs to be in the actual react component. Create a function variable with this in the AreaZeroData object 

```
const actionClick = () => {
                    stateContext.actions.toggleAreaPanel({ 
                        expanded: true 
                    });
}
```
Pass that to the <ZeroData>

Also, this is a class not a functional component. I think you'll need to use const stateContext = this.context

Reply:
i get this: `Uncaught TypeError: Cannot read property 'toggleAreaPanel' of undefined`

Maybe i need to pass `context` in as a prop

Reply:
Is it an arrow function?

Reply:
We can get on a call and debug. Also, you switched to this.context, yes?